website/source/docs/configuration/index.html.md

```diff
@@ -99,6 +99,7 @@ to specify where the configuration is.
 
 - `ui` `(bool: false, Enterprise-only)` – Enables the built-in web UI. Once
   enabled, the UI will be available to browsers at the standard Vault address.
+  This UI will be available on all listeners (address + port) at the `/ui` path.
```

Comment:
Maybe "Enables the built-in web UI, which is available on all listeners (address + port) at the `/ui` path. Browsers accessing the standard Vault API address will automatically redirect there."  -- sound good?